docs/standards/rest.md

```diff
@@ -115,7 +115,18 @@ headers:
 
 [Header field names are case insensitive](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2). Snyk v3 API specs must use kebab case for consistency. All non-standard headers that are unique to Snyk must begin with `snyk-` (e.g. `snyk-requested-version`).
 
-## <a id="request-parameters"></a>Request Parameters
+## <a id="reserved-request-parameters"></a>Reserved Request Parameters
+
+### <a id="filtering-authorized-access"></a>Filtering authorized access
+
+Some resources may allow filtering results based on authorizations granted to the authenticated principal. Some applications of this:
+
+- API applications: "Apply a policy to those resources the caller is an administrator of."
+- User interfaces: "Display editing widgets for those resources the user can edit."
+
+#### `has_permission` query parameter
```

Comment:
We might have a common schema type that defines these. However, it's also possible that some permissions are not relevant to some resources. In that case it might be reasonable for an endpoint to only declare a subset of those permissions.

Reply:
I think a convention for naming the slugs would be good enough, eg `org.read` -> `org_read`.